packages/amplify-category-auth/resources/cloudformation-templates/auth-template.yml.ejs

```diff
@@ -575,11 +575,13 @@ Resources:
             - '    ProviderDetails: {'
             - '     ''client_id'': providerCreds.client_id,'
             - '     ''client_secret'': providerCreds.client_secret,'
+            - '     ''oidc_issuer'': providerMeta.oidc_issuer,'
+            - '     ''attributes_request_method'': providerMeta.attributes_request_method,'
             - '     ''authorize_scopes'': providerMeta.authorize_scopes'
             - '    },'
             - '    ProviderName: providerMeta.ProviderName,'
             - '    UserPoolId: userPoolId,'
-            - '    AttributeMapping: providerMeta.AttributeMapping'
+            - '    AttributeMapping: providerMeta.attribute_mapping'
```

Comment:
We can not make this modification like this, because there are projects that are set up by older versions of the CLI, and in those project, it's named "AttributeMapping", the updated CLI will throw error on those old projects. 
If we decide to make this update, we have to add migration logic for those projects, which can be very messy and error prone.